src/libexpr/local.mk

```diff
@@ -26,7 +37,18 @@ endif
 # because inline functions in libexpr's header files call libgc.
 libexpr_LDFLAGS_PROPAGATED = $(BDW_GC_LIBS)
 
-libexpr_ORDER_AFTER := $(d)/parser-tab.cc $(d)/parser-tab.hh $(d)/lexer-tab.cc $(d)/lexer-tab.hh
+libexpr_ORDER_AFTER := $(d)/parser-tab.cc \
+                       $(d)/parser-tab.hh \
+                       $(d)/lexer-tab.cc \
+                       $(d)/lexer-tab.hh \
+                       $(d)/diagnostics.inc.hh \
+                       $(d)/diagnostics-id.inc.hh
+
+$(d)/diagnostics-id.inc.hh: $(d)/diagnostics-gen.nix $(d)/diagnostics.nix
+	$(trace-gen) $(nixcmd) --experimental-features "nix-command" eval --raw --file $< idmacros  > $@
```

Comment:
It seems like there are many problems caused by this line, because we want to open some directories in the sandbox-ed environment, should I do this?